.isort.cfg

```diff
@@ -1,2 +0,0 @@
-[settings]
-line_length = 132
```

Comment:
Why is line length being changed? I don't see anything in the PR descriptions about this so I am not following why this change is being made

Reply:
NVM I see why